### ISSUE TITLE
Look into collapsing validation reason codes

### ISSUE THREAD

**Issue by jeancochrane** [ccao-data/model-sales-val#116]
The list of reason codes for model determination seems like it may have some duplicates; in particular, it's unclear what the difference between "home flip" and "price swing" should be. We should double check the logic to see if these can be consolidated at all for the sake of simplicity.

**Comment by jeancochrane** [comment]
I took a look at this today and have some thoughts:

* **"Home flip sale" and "Price swing"** are substantively different, and I think the confusion is rooted in the naming rather than the logic. "Home flip sale" actually checks for short-term ownership plus high or low price, while "Price swing" looks for a large movement in price towards or away from the mean between two consecutive sales. I think both of these categories are useful, but the distinction between them would be clearer if "Home flip sale" were renamed to "Short-term owner".
* I'm not entirely convinced of the utility between the **"raw", "sqft", and "raw & sqft" versions of the high/low price flags**. They are also the only difference between the condo and res categories (condos only have naive high/low flags, with no specificity as to raw or sqft), so the logic would be simpler if we replaced them the more specific flags with naive high/low flags in the res case.
* The distinction between **"Anomaly" and the high/low price flags** feels unclear to me. Similar to the first bullet above, I wonder if a renaming from "Anomly" to "Statistical anomaly" would make it a little clearer; but perhaps someone with more context on the isolation forest algorithm would be able to clarify why both of these types of flags are necessary.
* **"PTAX-203 flag"** should probably be disaggregated into more specific categories, but I'm leaving that out of scope here in favor of https://github.com/ccao-data/model-sales-val/issues/118.

Taking this altogether, my suggestions for consolidation would be the following set of canonical model flags, leaving aside PTAX flags:

* Short-term owner (high price)
* Short-term owner (low price)
* Family sale (high)
* Family sale (low)
* Non-person sale (high)
* Non-person sale (low)
* Statistical anomaly (high)
* Statistical anomaly (low)
* Price swing (high)
* Price swing (low)
* Price (high)
* Price (low)

What do you think @dfsnow?

**Comment by jeancochrane** [comment]
Waiting on approval of the design doc to move forward with our design. I'll close this issue and open new ones for concrete next steps once the design gets approved.

**Comment by jeancochrane** [comment]
[Design doc](https://cookcounty-my.sharepoint.com/:w:/r/personal/jean_cochrane_cookcountyil_gov/Documents/Design%20Doc%20-%20Updated%20Sales%20Validation%20Specification.docx?d=wd6a2a31a0a664b168522078b16b3f92a&csf=1&web=1&e=znKgL9) is approved and work is queued up; closing.
